com.unity.robotics.urdf-importer/Tests/Runtime/UrdfComponents/UrdfJoints/UrdfJointTests.cs

```diff
@@ -0,0 +1,303 @@
+        Assert.AreEqual(UrdfJoint.JointTypes.Prismatic, UrdfJoint.GetJointType("prismatic"));
+        Assert.AreEqual(UrdfJoint.JointTypes.Planar, UrdfJoint.GetJointType("planar"));
+        Assert.AreEqual(UrdfJoint.JointTypes.Fixed, UrdfJoint.GetJointType("unknown"));
+    }
+
+    [Test]
+    public void GetAxis_JointAxis()
+    {
+        var axis = new RosSharp.Urdf.Joint.Axis(new double[] { 1, 2, 3 });
+        Assert.AreEqual(new Vector3(-2, 3, 1), TestUrdfJoint.Axis(axis));
+    }
+
+    [Test]
+    public void GetDefaultAxis_JointAxis()
+    {
+        Assert.AreEqual(new Vector3(-1, 0, 0), TestUrdfJoint.DefaultAxis());
+    }
+
+    [Test]
+    public void SetDynamics_Success()
+    {
+        var dynamics = new RosSharp.Urdf.Joint.Dynamics(1, 2);
+        GameObject linkObject = new GameObject("link");
+        var joint = linkObject.AddComponent<TestUrdfJoint>();
+        joint.Dynamics(dynamics);
+
+        var articulationBody = linkObject.GetComponent<ArticulationBody>();
+        Assert.AreEqual(1, articulationBody.linearDamping);
+        Assert.AreEqual(1, articulationBody.angularDamping);
+        Assert.AreEqual(2, articulationBody.jointFriction);
+
+        Object.DestroyImmediate(linkObject);
+    }
+
+    [Test]
+    public void SetDynamics_DefaultDynamics()
+    {
+        GameObject linkObject = new GameObject("link");
+        var joint = linkObject.AddComponent<TestUrdfJoint>();
+        joint.Dynamics(null);
+
+        var articulationBody = linkObject.GetComponent<ArticulationBody>();
+        Assert.AreEqual(0, articulationBody.linearDamping);
+        Assert.AreEqual(0, articulationBody.angularDamping);
+        Assert.AreEqual(0, articulationBody.jointFriction);
+
+        Object.DestroyImmediate(linkObject);
+    }
+
+    [Test]
+    public void ExportJointData_Success()
+    {
+        GameObject baseObject = new GameObject("base");
+        GameObject linkObject = new GameObject("link");
+        linkObject.transform.parent = baseObject.transform;
+        linkObject.transform.position = new Vector3(1, 2, 3);
+        linkObject.transform.rotation = Quaternion.Euler(4, 5, 6);
+
+        UrdfJoint.Create(baseObject, UrdfJoint.JointTypes.Fixed);
+        UrdfJoint.Create(linkObject, UrdfJoint.JointTypes.Revolute);
+        var joint = linkObject.GetComponent<UrdfJoint>().ExportJointData();
+
+        Assert.AreEqual(null, joint.name);
+        Assert.AreEqual("revolute", joint.type);
+        Assert.AreEqual("base", joint.parent);
+        Assert.AreEqual("link", joint.child);
+        Assert.AreEqual(new double[] { 3, -1, 2 }, joint.origin.Xyz);
+        UnityEngine.Assertions.Assert.AreApproximatelyEqual(-6 * Mathf.Deg2Rad, (float)joint.origin.Rpy[0]);
+        UnityEngine.Assertions.Assert.AreApproximatelyEqual(4 * Mathf.Deg2Rad, (float)joint.origin.Rpy[1]);
+        UnityEngine.Assertions.Assert.AreApproximatelyEqual(-5 * Mathf.Deg2Rad, (float)joint.origin.Rpy[2]);
```

Comment:
nit: do these need the `UnityEngine.Assertions` namespace?

Reply:
The normal assertions such as `Assert.AreEqual` are in the `NUnit.Framework` namespace. Because there are very minor differences in the float computation, I tried to use the `AreApproximateEqual` method provided in the `UnityEngine.Assertions` namespace, so I guess it is necessary here?